css/style.css

```diff
@@ -58,10 +62,11 @@ header img {
 .intro {
   background-image: url(/img/first-background.jpg);
```

Comment:
The image is not displaying. The css file is in a subfolder so your path needs to take this into account.

Reply:
> The image is not displaying. The css file is in a subfolder so your path needs to take this into account.

Hi Karen, I am not sure what the issue is because on my laptop the image is only being displayed if I set it with the path that I have already used.